webapp/channels/src/components/admin_console/license_settings/index.ts

```diff
@@ -43,7 +43,7 @@ function mapDispatchToProps(dispatch: Dispatch) {
             requestTrialLicense,
             openModal,
             getFilteredUsersStats,
-            getUsersLimits,
+            getUsersLimits: getAppLimits,
```

Comment:
0/5 if we want to rename also this prop.

Reply:
Done.